e2e/definitions/kind/config.yaml

```diff
@@ -5,3 +5,9 @@ nodes:
   extraMounts:
   - hostPath: ./e2e/debug/data
     containerPath: /tmp/e2e
+- role: worker
+  labels:
+    worker: true
+  extraMounts:
```

Comment:
Are the extraMounts actually necessary? They break the e2e-tests on my mac. Without them, they run mostly through. Except for the remark below.

Reply:
I forgot to update the worker/controlplane PVC. That's fixed now and works on my macOS. Can you check again? make sure to `make kind-clean` first.

Reply:
Werid, it still fails for me. But it's okay if it's working for you, I'll have a look at this another time.